src/main/java/com/github/manolo8/darkbot/Main.java

```diff
@@ -293,8 +297,7 @@ private void tickLogic(boolean running) {
                     setRunning(false);
                 } catch (Throwable e) {
                     featureRegistry.getFeatureDefinition(behaviour)
-                            .getIssues()
-                            .addFailure("bot.issue.feature.failed_to_tick", IssueHandler.createDescription(e));
+                            .addIssue(e, "bot.issue.feature.failed_to_tick", PluginIssue.Level.ERROR);
```

Comment:
i generally dislike that we're using addIssue with a level everywhere, i think addFailure/addWarning/addInfo are better for readability.

Now that we have lombok it should be pretty easy to `@Delegate` the issues in FeatureDefinition, and make FeatureInfo interface extend Issues interface, on an API level.
What that essentially means is all "getIssues().*" methods can now be called directly on FeatureDefinition